### PR TITLE
Condense the layout of the jod details page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Condense the layout of the job details page

## QA
- Open the demo
- Go to /careers/2383894/vp-cloud-alliances-remote
- See that there is more content above the fold than [on live](https://canonical.com/careers/2383894/vp-cloud-alliances-remote)
- Check it looks ok on small screens

## Screenshots

| Before | After |
| --- | --- |
| image | image |

### CODE REVIEW

**Review by nottrobin — APPROVED** [review]
Lgtm

File: templates/careers/job-detail.html [hunk 1]
```diff
@@ -45,73 +39,60 @@ <h1>Careers</h1>
...
+
+  <div class="col-5 col-start-large-8 p-card--strip">
+    <h3>Apply for this role</h3>
+    <form action="{{ request.path }}" method="POST" enctype="multipart/form-data" class="js-roles-list--form" onsubmit="dataLayer.push({
+          'event': 'GAEvent',
+          'eventCategory': 'Form',
+          'eventAction': 'job application: {{ job.id }}',
+          'eventLabel': 'Submit Application',
+          'eventValue': undefined });" >
+      <fieldset style="border: none; margin-bottom: 0;">
```
Comment (carkod) [inline templates/careers/job-detail.html:58]: `u-no-margin--bottom`? I am not sure if we have something for border maybe we should?

**Comment by lyubomir-popov** [comment]
Is there a reason the fieldset is styled differently compared to the [vanilla default](https://vanillaframework.io/docs/examples/base/forms/fieldset)?

**Comment by carkod** [comment]
Also when I enter a wrong format for email address it returns `Unprocessable entity` which I think it's not very friendly error message. Maybe we should put in the validation JS that we have on ubuntu.com? But I can file an issue for that if you prefer

**Comment by waynecrosby** [comment]
@anthonydillon Can we make the title go the full 12cols width, it's still wrapping in the left hand column so for long titles it knocks the form down?

image

**Comment by anthonydillon** [comment]
> Is there a reason the fieldset is styled differently compared to the vanilla default?

@lyubomir-popov totally agree but I didn't change that just moved its position.

File: templates/careers/job-detail.html (hunk 1 above)
Comment (anthonydillon) [reply templates/careers/job-detail.html:58]: Ideally we would code up a true pattern in Vanilla but that will be in the future.

**Comment by anthonydillon** [comment]
@waynecrosby good point! Done.
image

**Comment by waynecrosby** [comment]
@anthonydillon @lyubomir-popov Looks good. The form was originally designed to be like the below, I am assuming it's because its pulled from Greenhouse into our page that we get the strange box-in-box effect? Happy to revert to the simple Vanilla standard white box with outline.

image

**Pull request merged**